src/AEPAssuranceiOSUnitTests/mono_crash.11a67480d0.0.json

```diff
@@ -0,0 +1,731 @@
+{
```

Comment:
This is a test file?

Reply:
These are files that contain mono crash dump information... It is safe for them to be deleted an not checked In
Let me do that.. thanks for identifying them